utils/progressbar/filesprogressbar.go

```diff
@@ -185,8 +185,11 @@ func (p *filesProgressBarManager) RemoveProgress(id int) {
 	defer p.barsWg.Done()
 	defer p.barsRWMutex.RUnlock()
 	p.bars[id-1].Abort()
-	p.generalProgressBar.Increment()
+}
 
+// Increases general progress bar by 1
```

Comment:
I think we can remove this comment, it restates the obvious and provides no new information.